lib/ansible/module_utils/k8svirt/helper.py

```diff
@@ -379,3 +395,81 @@ def replace(self, body, namespace, name):
         return (self.__client.
                 replace_namespaced_virtual_machine_instance_preset(
                     vmips_body, namespace, name))
+
+
+class PersistentVolumeClaimHelper(object):
+    """ Helper class for PersistentVolumeClaim resources """
+
+    def __init__(self, client):
+        """ Class constructor """
+        self.__client = client
+
+    def create(self, body, namespace, wait=False, timeout=300, interval=3):
+        """ Create PersistentVolumeClaim resource """
+        pvc_body = core_sdk.V1PersistentVolumeClaim().to_dict()
+        pvc_body.update(copy.deepcopy(body))
+        created_pvc = self.__client.create_namespaced_persistent_volume_claim(
+            namespace=namespace, body=pvc_body)
+        if not wait:
+            return created_pvc
+
+        name = pvc_body['metadata']['name']
+        start = time.time()
+        while time.time() < start + timeout:
```

Comment:
Why not to use a watch here? In similar way we do it [here](https://github.com/kubevirt/client-python/blob/master/examples/example_vms.py#L34)